### ISSUE TITLE
Syncronize views

### ISSUE THREAD

**Issue by hibikutek** [notepadqq/notepadqq#693]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ X ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/notepadqq/notepadqq/blob/master/CONTRIBUTING.md#question
```
Having used notepad++ quite a lot, one feature that I would like to see in notepadqq is the ability to syncronize your views so that scrolling will scroll both views at the same time.  Optimally I would like it to have a toggle-able menu option with a keyboard shortcut that I can enable, scroll both views together, and then toggle off.

If this is something that could better be done as an extension, I have a bit of drive to develop that myself, but I feel like the api is a bit lacking on 'views'.

**Comment by danieleds** [comment]
Yeah the API is not very expressive right now. We're working on it.
